### ISSUE TITLE
🐞 Bug on IE 11 - Syntax error on version 3.*.* (not present before in 2.*.*)

### ISSUE THREAD

**Issue by LuXDAmore** [aws-amplify/amplify-js#5405]
🐞 **Describe the bug**
I'm using the `@aws-amplify/auth` and `@aws-amplify/core` packages to add the AWS login functionality to a website.

Everything is fine in the previous releases:
- `@aws-amplify/auth: ^2.1.8`
- `@aws-amplify/core: ^2.3.0`

But, updating to the new one:
- `@aws-amplify/auth: ^3.2.3`
- `@aws-amplify/core: ^3.2.3`

**Cause (only in IE 11) a _Syntax error_ in the console.**

P.S.: I've tried to update the libraries one-by-one, but the issue still persist.

___

**To Reproduce**

1. Install the latest version of `@aws-amplify/auth@^3.2.3` or `@aws-amplify/core@^3.2.3`;
2. Import it in a project (i'm using it with webpack, in a Vue/Nuxt environment);
3. Open the IE 11 Browser;
4. Check the console.

**Device**

- Device: PC
- OS: Windows 10 x64
- Browser: Chrome 83.0.4103.7 and IE 11

**Sample code**

```javascript

import Auth from '@aws-amplify/auth';
import Amplify from '@aws-amplify/core';

Amplify.configure(
    {
         Auth: config,
     }
);

Auth.currentSession();

```
![2020-04-14](https://user-images.githubusercontent.com/3841171/79230999-b6afe980-7e65-11ea-87b3-d9a3fea61d3f.png)

**Comment by ericclemmons** [comment]
Awaiting a version bump for https://www.npmjs.com/package/@aws-sdk/eventstream-serde-browser

**Comment by slavco86** [comment]
@LuXDAmore 
I have found a workaround to transpile dependencies till we can get a fix from @ericclemmons , hopefully soon.

So the workaround is:

**vue.config.js**

```
module.exports = {
  transpileDependencies: ["@aws-sdk", "fast-xml-parser"]
}
```
**babel.config.js**
Now, the babel config - I'm not saying you have to have it all like this - this is just mine. All I had to change was `useBuiltIns` from `usage` to `entry`, so it might be that it's all that is needed. But you might also need to be aware of what presets I am actually using, which is different to what comes with Vue-CLI out of the box.

```module.exports = {
  plugins: [
    "@babel/proposal-object-rest-spread",
    "@babel/plugin-syntax-dynamic-import"
  ],
  presets: [
    [
      "@babel/preset-env",
      {
        targets: {
          ie: "11",
          browsers: "last 2 versions"
        },
        useBuiltIns: "entry",
        corejs: { version: 3, proposals: true }
      }
    ]
  ]
};
```

**main.js**
```
import "core-js/stable";
import "regenerator-runtime/runtime";
```

**Comment by LuXDAmore** [comment]
Thank You @slavco86 , but unfortunately i can't touch my `babel.config` file 😆

**Comment by LuXDAmore** [comment]
@sammartinez Thank You very much for your support! 🚀 

I've updated to the latest versions:
> `@aws-amplify/auth: ^3.2.13`
> `@aws-amplify/core: ^3.3.3`

But unfortunately the same issue still persist on IE 11 😭

**Comment by sammartinez** [comment]
@LuXDAmore When you tried the latest version, did you remove your lock files and re installed?

**Comment by LuXDAmore** [comment]
@sammartinez Yes, i've also tried to deleted the entrire `node_modules` folder (just to be sure to have everything up-to-date).

**Comment by LuXDAmore** [comment]
@sammartinez 

IDK if this can help, but in my `package-lock.json` i can see the **gamma-version** used everywhere, except for the `@aws-sdk/client-cognito-identity` that is still requiring an **alpha-version** of the `@aws-crypto` libraries.

```json

{
    "@aws-sdk/middleware-serde": {
        "version": "1.0.0-gamma.1",
        "resolved": "https://registry.npmjs.org/@aws-sdk/middleware-serde/-/middleware-serde-1.0.0-gamma.1.tgz",
        "integrity": "sha512-4rhB/x8zGmiN1k12fPG1LOxvK41XE8yOlayQyach2Lct30i38Oel/7wpnYaIXuTGFaY+npC127fgAY3XuMcdcw==",
        "requires": {
            "@aws-sdk/types": "1.0.0-gamma.1",
            "tslib": "^1.8.0"
        }
    },
    "@aws-sdk/client-cognito-identity": {
        "version": "1.0.0-gamma.2",
        "resolved": "https://registry.npmjs.org/@aws-sdk/client-cognito-identity/-/client-cognito-identity-1.0.0-gamma.2.tgz",
        "integrity": "sha512-gGwTrKRN+mj48m4tXTilTbp1/aYO5mGIBPocDsG7digNFqxmjzFVNCq/yJfoBg7MCNBVlOFJ7i3bxM6H4cylow==",
        "requires": {
            "@aws-crypto/sha256-browser": "^1.0.0-alpha.0",
            "@aws-crypto/sha256-js": "^1.0.0-alpha.0"
        }
    }
},

```

**Comment by sammartinez** [comment]
@Amplifiyer Any idea on this?

**Comment by KwFung7** [comment]
aws/aws-sdk-js-v3#1319
The issue should be fixed in latest aws sdk v1.0.0-gamma.3 release (https://github.com/aws/aws-sdk-js-v3/releases/tag/v1.0.0-gamma.3

**Comment by sammartinez** [comment]
> [aws/aws-sdk-js-v3#1319](https://github.com/aws/aws-sdk-js-v3/issues/1319)
> The issue should be fixed in latest aws sdk v1.0.0-gamma.3 release (https://github.com/aws/aws-sdk-js-v3/releases/tag/v1.0.0-gamma.3)

Correct, we are testing this internally from the other changes within the release to validate it does not break any other additional Amplify use cases. We have an ETA of next week to get this change out. Thanks for the feedback!

**Comment by LuXDAmore** [comment]
@sammartinez Any news?

**Comment by petrovicigor** [comment]
> @sammartinez Any news?

Try Amplify 3.0.21

**Comment by petrovicigor** [comment]
> @sammartinez Any news?

Try Amplify 3.0.21

**Comment by LuXDAmore** [comment]
@petrovicigor `@aws-amplify/auth: 3.0.21` and `@aws-amplify/core: 3.0.21`?

Because with [this one](https://github.com/aws-amplify/amplify-js/issues/5405#issuecomment-642529928) fail

**Comment by sammartinez** [comment]
This issue is resolved in the latest release, 3.0.22
